tests/test_smoke_eval.py

```diff
@@ -0,0 +1,120 @@
+from pathlib import Path
+
+from elk import Extract
+from elk.evaluation import Eval
+from elk.extraction import PromptConfig
+from elk.training import CcsReporterConfig, EigenReporterConfig
+from elk.training.train import Elicit
+from tests.utils import *
+
```

Comment:
seems you have some uncommited file utils

Reply:
whoops, I just pushed!